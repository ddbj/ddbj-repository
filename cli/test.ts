import { snapshotTest } from 'cliffy/testing/mod.ts';

import mainCommand from './main_command.ts';

await snapshotTest({
  name: '--help',
  meta: import.meta,
  denoArgs: ['--allow-all'],

  async fn() {
    await mainCommand.parse(['--help']);
  }
});
