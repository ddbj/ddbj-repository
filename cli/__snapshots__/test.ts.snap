export const snapshot = {};

snapshot[`--help 1`] = `
stdout:
"
Usage:   ddbj-repository
Version: 0.1.0          

Description:

  Command-line client for DDBJ Repository API

Options:

  -h, --help     - Show this help.                            
  -V, --version  - Show the version number for this program.  

Commands:

  auth        -                            
  validation  - Manage validations.        
  submission  - Manage submissions.        
  completion  - Generate shell completions.

Environment variables:

  DDBJ_REPOSITORY_API_URL  <url>  - API endpoint URL  

"
stderr:
""
`;
